src/Core/Services/Implementations/PolicyService.cs

```diff
@@ -53,25 +54,52 @@ public class PolicyService : IPolicyService
                 var currentPolicy = await _policyRepository.GetByIdAsync(policy.Id);
                 if (!currentPolicy?.Enabled ?? true)
                 {
-                    if (currentPolicy.Type == Enums.PolicyType.TwoFactorAuthentication)
+                    var kickableOffenses = new List<Enums.PolicyType>() {
```

Comment:
formatting here. I think you need to drop the `()`

Reply:
done